ds3-autogen-python/src/main/resources/tmpls/python/commands/all_commands.ftl

```diff
@@ -48,8 +69,7 @@ class AbstractResponse(object):
   def __check_status_codes__(self, expected_codes):
     if self.response.status not in expected_codes:
       err = "Return Code: Expected %s - Received %s" % (expected_codes, self.response.status)
-      ds3error = XmlSerializer().to_ds3error(self.response.read())
-      raise RequestFailed(err, ds3error)
+      raise RequestFailed(err, self.response)
```

Comment:
So we're not really parsing it at all.  We should look enhancing that in the future.